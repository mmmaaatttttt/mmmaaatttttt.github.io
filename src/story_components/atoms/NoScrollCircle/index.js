import styled from "styled-components";

const NoScrollCircle = styled.circle`
  touch-action: none;
  transition: stroke-width 300ms;

  &:hover,
  &:active {
    fill: ${props => props.stroke};
    stroke-width: ${props => props.r * 1.5};
    cursor: pointer;
  }
`;

export default NoScrollCircle;