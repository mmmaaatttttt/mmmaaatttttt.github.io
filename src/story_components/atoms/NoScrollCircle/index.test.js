import React from "react";
import { render } from "@testing-library/react";
import NoScrollCircle from "./";

it("renders without crashing", () => {
  render(<NoScrollCircle />);
});

it("matches snapshot with props", () => {
  const { asFragment } = render(<NoScrollCircle stroke="blue" />);
  expect(asFragment()).toMatchSnapshot();
});
