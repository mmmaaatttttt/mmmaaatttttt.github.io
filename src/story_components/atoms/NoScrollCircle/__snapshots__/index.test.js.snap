// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot with props 1`] = `
<DocumentFragment>
  <circle
    class="sc-bdVaJa iGYVAK"
    stroke="blue"
  />
</DocumentFragment>
`;
